### ISSUE TITLE
[Enhancement] Device type: Button

### ISSUE THREAD

**Issue by Trifenix** [t0bst4r/home-assistant-matter-hub#341]
### Requested feature

Would be nice to have support for Ring intercom and his button domanin

### Documentation & Issues

- [X] I have reviewed the list of [currently supported entities and their limitations](https://github.com/t0bst4r/home-assistant-matter-hub?tab=readme-ov-file#5-supported-domains--device-types).
- [X] I have searched the [issues](https://github.com/t0bst4r/home-assistant-matter-hub/issues) for a similar requests.
- [X] I have [checked](https://github.com/project-chip/matter.js/blob/main/docs/ECOSYSTEMS.md) if my controller is known to support the requested device type.

**Comment by zepab** [comment]
> [edit] : would the fix done at #195 works too for your case ?

answering to my self, nope, #195 is for input_button, not button.

**Comment by gbertsch** [comment]
Agreed, any plans to support button type? Looking to expose my homelink button from tesla_fleet integration.

**Comment by zepab** [comment]
@t0bst4r I tried and validate a change into https://github.com/t0bst4r/home-assistant-matter-hub/pull/480/ 
This was the first time for me with hass add-ons dev and deployment.  May be I did it wrong,  do no hesitate to tell me.

This was to be able to trigger my gate (a NICE motorisation seen over devmel airsend integration). 

of course saying open or close is doing the same thing because it is like pressing the remote controller button that would either open/stop/close the gate.

**Comment by gbertsch** [comment]
Appreciate that, I will give it a test.

May want to add support in Supported Device Types table as well.

**Comment by zepab** [comment]
> Appreciate that, I will give it a test.
> 
> May want to add support in Supported Device Types table as well.

I just commit with --amend and push forced into my branch for the readme so commit here is fine https://github.com/zepab/home-assistant-matter-hub/commit/ab7ab7a8aa2eec50c52854e1a262a31e9e9f2733 
but it seems you already merge it. so fast :-).
